README.md

```diff
@@ -48,17 +54,33 @@ eyre = "0.4"
 color-eyre = { version = "0.3", default-features = false }
 ```
 
-## Example
+## Features
+
+### Multiple report format verbosity levels
+
+`color-eyre` provides 3 different report formats for how it formats the captured `SpanTrace`
+and `Backtrace`, minimal, short, and full. Take the following example, taken from
```

Comment:
nit: the correct punctuation to start a list of options is a colon, not a comma. Also in formal writing, numbers are usually spelt out in full.

```suggestion
`color-eyre` provides three different report formats for how it formats the captured `SpanTrace`
and `Backtrace`: minimal, short, and full. Take the following example, taken from
```